kong/tracing/instrumentation.lua

```diff
@@ -236,7 +236,7 @@ function _M.request(ctx)
       ["http.host"] = host,
       ["http.scheme"] = scheme,
       ["http.flavor"] = http_flavor,
-      ["net.peer.ip"] = client.get_ip(),
+      ["net.peer.ip"] = client.get_forwarded_ip(),
```

Comment:
I think Kong using `["net.peer.ip"] = client.get_ip(),` is Okay for the Kong use case, if you want to add `forward` IP attribute to tracing instrumentation PDK, Maybe you can add a new entry like `["net.peer.forwarded_ip"]`, but I am not sure if this is Okay for tracing system.

Reply:
updated and pushed